BinaryEncoding.md

```diff
@@ -369,10 +351,11 @@ a `data_segment` is:
 
 ### Name section
 
-ID: `name`
+Section string: `"name"`
 
-The names section does not change execution semantics and a validation error in
-this section does not cause validation for the whole module to fail and is
+The names section does not change execution semantics, and thus is not allocated
+a section opcode.
+A validation error in this section does not cause validation for the whole module to fail and is
 instead treated as if the section was absent. The expectation is that, when a
```

Comment:
Could you move this sentence up above, below the para that introduces "user define sections", changing "this section" to "a user defined section"?  That way it's clear that this is true for all user-defined sections, not just the names section.